test/unit/MemberRoles/changeDependentContractAddress.js

```diff
@@ -0,0 +1,45 @@
+const { InternalContractsIDs } = require('../utils').constants;
+const { expect } = require('chai');
+const { hex } = require('../../../lib/helpers');
+const { ZERO_ADDRESS } = require('../../../lib/constants');
+
+describe('changeDependentContractAddress', function () {
+  before(async function () {
+    const { quotationData, memberRoles } = this.contracts;
+    const { governanceContracts, defaultSender } = this.accounts;
+    await quotationData.connect(governanceContracts[0]).setKycAuthAddress(defaultSender.address);
+    await memberRoles.connect(governanceContracts[0]).setKycAuthAddress(quotationData.address);
+  });
+
+  it('should change authorized address for the role', async function () {
+    const { memberRoles, master } = this.contracts;
+
```

Comment:
Given there's only one test here, move the contents of the `before` in the test itself